manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml

```diff
@@ -261,6 +261,7 @@ spec:
                 - "--leader-election-id=kiali-operator"
                 - "--watches-file=./$(WATCHES_FILE)"
                 - "--health-probe-bind-address=:6789"
+                - "--metrics-bind-address=:8080"
```

Comment:
NOTE: it is OK changing this for the OSSM OLM metadata because this option exists in earlier versions, too. This includes the base image uses downstream, as you see verified below:

4.15:
```
$ podman run  -it --rm registry.redhat.io/openshift4/ose-ansible-operator:v4.15 --help | grep metrics-bind-address
      --metrics-bind-address string    The address the metric endpoint binds to (default ":8080")
```

4.17:
```
$ podman run  -it --rm registry.redhat.io/openshift4/ose-ansible-operator:v4.17 --help | grep metrics-bind-address
      --metrics-bind-address string    The address the metric endpoint binds to (default ":8080")
```

But notice the default of these older versions is 8080, which is what we did not expect to change but did in newer versions of the upstream base image:

```
$ podman run -it --rm quay.io/operator-framework/ansible-operator:v1.35.0 --help | grep metrics-bind-address
      --metrics-bind-address string    The address the metric endpoint binds to (default ":8080")

$ podman run -it --rm quay.io/operator-framework/ansible-operator:v1.36.0 --help | grep metrics-bind-address
      --metrics-bind-address string    The address the metric endpoint binds to (default ":8443")
```